vars/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+openvpn_pki_password: "{{ lookup('env','OPENVPN_PKI_PASSWORD') }}"
```

Comment:
Actually what we need to do is not to check if the OPENVPN_PKI_PASSWORD env var is set, as it's an end-user implementation detail, but to check if the pki_password ansible var is defined.
Please check: https://github.com/ebarault/ovenvpn-ansible-role/blob/otp/defaults/main.yml#L5

Reply:
I think i see your point. new patch incoming.